packages/react-i18n/src/constants.ts

```diff
@@ -22,6 +23,13 @@ export const dateStyle = {
     day: '2-digit',
     year: 'numeric',
   },
+  [DateStyle.HumanizeWithTime]: {
+    month: 'long',
+    day: '2-digit',
```

Comment:
Can we make sure all days in this file use numeric vs 2-digit? Trying to get rid of those leading zeroes